packages/gasket-utils/lib/package-manager.js

```diff
@@ -78,19 +70,19 @@ class PackageManager {
     const env = Object.assign({}, process.env);
 
     if (this.manager === 'npm') {
-      //
       // Given that we are spawning `npm` in a child process a number of
       // valid TTY options need to be disabled to get proper output
       //
       // Remark: should we make this the default in ./npm?
-      //
       const argv = [
-        '--loglevel', 'info',
-        '--spin', 'false',
-        '--progress', 'false'
+        '--loglevel',
+        'info',
+        '--spin',
+        'false',
+        '--progress',
+        'false'
```

Comment:
This was easier to understand with flag/value in same line

Reply:
Great feedback! Will update!